baseplate/_utils.py

```diff
@@ -42,3 +44,84 @@ def __get__(self, obj, type=None):
         ret = self.wrapped(obj)
         setattr(obj, self.wrapped.__name__, ret)
         return ret
+
+
+SerializedBatch = collections.namedtuple("SerializedBatch", "count bytes")
+
+# maximum time (seconds) an event can sit around while we wait for more
+# messages to batch
+MAX_BATCH_AGE = 1
+# maximum size (in bytes) of a batch of events
```

Comment:
these constants are event-specific, can they be in the events module only? (it makes sense for the base to not have any default since various users would have different needs)

(V1Batch/TraceBatch can specify the default for their use cases)